scripts/mkdir.bat

```diff
@@ -0,0 +1,4 @@
+@echo off
+setlocal enableextensions
+md %1
```

Comment:
unfortunate that we need this. is this really only for Github Actions?

Reply:
I actually think we can remove this if we aren't making any directories in the makefile

Reply:
We will be once merged. Worth dropping until needed?

Reply:
I don't think so. the MKDIR commands will only work recursively on windows via this script. We do use mkdir when we compile the project. Removing this script breaks the windows compile step in CICD. Strangely enough this isn't an issue on my local machine. 

I say we cop this for now. It isn't a fun solution but it's the best one I can find at the moment.